examples/web-instrumentation/README.md

```diff
@@ -0,0 +1,384 @@
+# Instrumenting Web Applications End-to-End with Stackdriver and OpenTelemetry
+
+The app is something like a simple web version of Apache Bench. It includes
+JavaScript browser code that drives HTTP requests to a Node.js backend that
+can be run anywhere that you can run Node.
+
+Create a Google Cloud Project with billing enabled. Install Node.js.
+Install the Google Cloud SDK.
+
+## Setup
+
+Clone this repository to your environment with the command
+
+```shell
+git clone https://github.com/GoogleCloudPlatform/professional-services.git
+```
+
+Change to this directory and set an environment variable to remember the
+location
+
+```shell
+cd professional-services/examples/web-instrumentation
+WI_HOME=`pwd`
+```
+
+Set Google Cloud SDK to the current project 
+
+```shell
+export GOOGLE_CLOUD_PROJECT=[Your project]
+gcloud config set project $GOOGLE_CLOUD_PROJECT
+```
+
+Enable the required services
+
+```shell
+gcloud services enable bigquery.googleapis.com \
+  cloudbuild.googleapis.com \
+  cloudtrace.googleapis.com \
+  compute.googleapis.com \
+  container.googleapis.com \
+  containerregistry.googleapis.com \
+  logging.googleapis.com \
+  monitoring.googleapis.com
+```
+
+
+Install the JavaScript packages required by both the server and the browser:
+
+```shell
+npm install
+```
+
+## OpenTelemetry collector
+
+Open up a new shell to clone the OpenTelemetry collector contrib project, which
+contains the Stackdriver exporter
+
+```shell
+git clone https://github.com/open-telemetry/opentelemetry-collector-contrib
```

Comment:
Do they need to be in a new directory when they run this command?

Reply:
Yes, I reworded the text above to make that clear